2018/_includes/hotels.html

```diff
@@ -4,33 +4,7 @@
         <div class="widget-title"><h4>Hotels</h4></div>
         <div class="widget-content">
             <div class="eventstation-latest-posts-widget">
-                <ul>
```

Comment:
Instead of removing all the behaviour with the list of hotels... what about empty the JSON file?

Reply:
you are right, my bad :sweat: